README.md

```diff
@@ -48,7 +48,7 @@ more. See [the wiki](https://github.com/neovim/neovim/wiki/Installing-Neovim)!
 
 ### Community
```

Comment:
Let's remove this section entirely and put the Community link immediately right of the "Website" link at the top.